src/stylus/cozy-ui/app/layouts.styl

```diff
@@ -27,6 +27,15 @@ $app-2panes-toolbar
         display  flex
         flex-direction column
 
+    .drawer
+        display flex
+        flex-direction column
+        padded .5em
```

Comment:
I prefer don't add spacers in layouts mixins, because in most cases you'll want to deal with it app-by-app, depending of your design and features.